src/nidcpower/metadata/enums.py

```diff
@@ -847,6 +853,18 @@
                 },
                 'name': 'NIDCPOWER_VAL_OUTPUT_CONSTANT_CURRENT',
                 'value': 1
+            },
+            {
+                'name': 'NIDCPOWER_VAL_OUTPUT_OVER_VOLTAGE',
```

Comment:
We may not want these additional enum values. But I do think they're legitimate enum values for the "OutputStates" enum so maybe these additions are okay here.